### PR TITLE
[18.09] Make engine scope a build time setting

### PR DESCRIPTION
**Pull request by dhiltgen**
This should make it easier to change downstream builds
to change the scope.

Signed-off-by: Daniel Hiltgen <daniel.hiltgen@docker.com>

### CODE REVIEW

File: rpm/Makefile [hunk 1]
```diff
@@ -114,7 +115,11 @@ rpmbuild/SOURCES/docker.service: ../systemd/docker.service
 
 rpmbuild/SOURCES/dockerd.json: ../common/dockerd.json
 	mkdir -p $(@D)
-	sed -e 's!$${ENGINE_IMAGE}!$(ENGINE_IMAGE)!' -e 's/$${IMAGE_TAG}/$(IMAGE_TAG)/' $< > $@
+	sed -e \
+	    's!$${ENGINE_IMAGE}!$(ENGINE_IMAGE)!' \
+	    's!$${ENGINE_SCOPE}!$(ENGINE_SCOPE)!' \
```
Comment (seemethere) [inline rpm/Makefile]: you're missing a `-e`

Comment (dhiltgen) [reply rpm/Makefile]: crap - good catch.

Comment (dhiltgen) [reply rpm/Makefile]: fixed

**Comment by seemethere** [comment]
@jose-bigio we'll also need to forward port this to master

**Pull request merged**